test/NuGet.Clients.Tests/NuGet.SolutionRestoreManager.Test/RestoreOperationLoggerTests.cs

```diff
@@ -12,17 +13,21 @@ namespace NuGet.SolutionRestoreManager.Test
     [Collection(MockedVS.Collection)]
     public class RestoreOperationLoggerTests
     {
+        public RestoreOperationLoggerTests(GlobalServiceProvider sp)
+        {
+            sp.Reset();
+        }
+
         [Fact]
         public async Task StatusBarProgress_StartAsync_CancellationTokenThrowsAsync()
         {
             // Prepare
-            var cts = new CancellationTokenSource();
+            var token = new CancellationToken(canceled: true);
```

Comment:
Flakey test fix. Sometimes the Assertion ran before the task got cancelled. Shoutout to @dominoFire for spotting this super quickly! ⚡